src/components/transcript/index.tsx

```diff
@@ -60,6 +67,58 @@ export type sideBarContentUpdateParams<T, K> = {
   name: K;
 };
 
+const getTranscriptContent = (content: TranscriptContent) => {
+  const {
+    speakers,
+    categories,
+    tags,
+    title = "",
+    loc = "",
+    date,
+    ...restContent
+  } = content;
+  const data: SideBarData = {
+    list: {
+      speakers: reconcileArray(speakers),
+      categories: reconcileArray(categories),
+      tags: reconcileArray(tags),
+    },
+    text: {
+      title,
+    },
+    loc: {
+      loc,
+    },
+    date: {
+      date: date ? new Date(date) : null,
+    },
+  };
+
+  for (const field of Object.keys(
+    omit(restContent, ["body", "media", "transcript_by"])
```

Comment:
I see that we are omitting those fields (`media` and `transcript_by`) in a couple of places in the codebase. 
Isn't it simpler to just remove them at the start of our processing logic so we don't have to deal with them everywhere? This also makes our code less error-prone.

I believe that if you filter for those in the `getUpdatedContent` and `getTranscriptContent` method we wouldn't need to remember to omit them at different places.

Reply:
@BalogunofAfrica what about this one?

Reply:
Added it as well